.azure-pipelines/run-test-elastictest-template.yml

```diff
@@ -202,7 +216,7 @@ steps:
         --kvm-build-id $(KVM_BUILD_ID) \
         --kvm-image-branch "${{ parameters.KVM_IMAGE_BRANCH }}" \
         --deploy-mg-extra-params="${{ parameters.DEPLOY_MG_EXTRA_PARAMS }}" \
-        --common-extra-params="${{ parameters.COMMON_EXTRA_PARAMS }}" \
+        --common-extra-params="${{ parameters.COMMON_EXTRA_PARAMS }} ${PTF_IMAGETAG}" \
```

Comment:
@lerry-lee Can you confirm if the `PTF_IMAGETAG` argument will be used by `restart-ptf` if it is supplied here?

Reply:
Currently we don't support specifying PTF_IMAGETAG param in restart-ptf step, we will support it first and then update here to explain the usage.

Reply:
@lerry-lee if you have an issue to track that can you please reference that here?

Reply:
@opcoder0 yes, https://github.com/sonic-net/sonic-mgmt/pull/14516